src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Logging/LogHubLogger.cs

```diff
@@ -37,23 +37,30 @@ public LogHubLogger(string categoryName, LogHubLogWriter feedbackFileLogWriter)
 
         public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception exception, Func<TState, Exception, string> formatter)
         {
-            var formattedResult = formatter(state, exception);
+            try
```

Comment:
This is the future proofing, to prevent a bad log message bringing down an actual feature.